gnmpagerduty/tasks.py

```diff
@@ -0,0 +1,181 @@
+import logging
+from celery.schedules import crontab
+from celery.decorators import periodic_task
+from celery import Celery
+
+app = Celery()
+
+
+log = logging.getLogger(__name__)
+
+logger = logging.getLogger('main')
```

Comment:
I think the reason way because bits of this code where copied from places where they worked and I did not want to break things by alerting them.